### ISSUE TITLE
Please bring back single-target commands

### ISSUE THREAD

**Issue by kevinfoley** [eezstreet/SWATEliteForce#231]
"Disarm all", "secure all", "search and secure" are convenient but sacrifice too much tactical control. Sometimes I need finer control over what my officers are doing. E.g. if one suspect is in a corner and another is next to an open doorway, I might want the team to secure the suspect in the corner but not risk standing in the doorway to secure the other suspect.

Hopefully it is possible to have both single-target and all-target commands. If not, I'd much prefer the old system.

EDIT: It seems like the single-target commands are still supported using speech commands, which is nice, but people who use the old style command system are out of luck.

**Comment by kevinfoley** [comment]
Also, on that note, if you issue a search-and-secure but there is nothing to secure, the team gives an affirmative but doesn't move or do anything. This is very jarring and feels unpolished compared to the other commands. It would be nice if they either played the "no evidence here" VO or at least walked around the room a little.

**Comment by sandman332** [comment]
I second this.  Its nice to have the all commands, but there are times I want them to just do the single version of it.

**Comment by eezstreet** [comment]
It is possible to do this if you are aiming at the target and press space (they are still the default contextual option)
________________________________

**Comment by eezstreet** [comment]
Should this be changed so it's available on the regular menu as well, or left as a context-only option?

**Comment by sandman332** [comment]
I feel like the regular menu would be good.

**Comment by smug** [comment]
I don't think it should be added. Why clutter the menu when you already get single target commands from the context action?

**Comment by eezstreet** [comment]
There isn't enough room in the menu to support this in all contexts, and it is the default option on SPACE when looking at a target in the CCI and GCI. Removing.
